site/man/rabbitmq-streams.8.html

```diff
@@ -303,7 +314,155 @@ <h3 class="Ss" title="Ss" id="Stream_plugin"><a class="permalink" href="#Stream_
     <div class="Bd Bd-indent lang-bash"><code class="Li">rabbitmq-streams
       list_stream_publishers connection_pid publisher_id stream</code></div>
   </dd>
+  <dt><a class="permalink" href="#add_super_stream"><code class="Cm" title="Cm" id="add_super_stream">add_super_stream</code></a>
+    <var class="Ar" title="Ar">super-stream</var>
+    [<div class="Op"><code class="Fl" title="Fl">--vhost</code>
+    <var class="Ar" title="Ar">vhost</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--partitions</code>
+    <var class="Ar" title="Ar">partitions</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--routing-keys</code>
+    <var class="Ar" title="Ar">routing-keys</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--max-length-bytes</code>
+    <var class="Ar" title="Ar">max-length-bytes</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--max-age</code>
+    <var class="Ar" title="Ar">max-age</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--stream-max-segment-size-bytes</code>
+    <var class="Ar" title="Ar">stream-max-segment-size-bytes</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--leader-locator</code>
+    <var class="Ar" title="Ar">leader-locator</var></div>]
+    [<div class="Op"><code class="Fl" title="Fl">--initial-cluster-size</code>
+    <var class="Ar" title="Ar">initial-cluster-size</var></div>]</dt>
+  <dd>
+    <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">super-stream</var></dt>
+      <dd>The name of the super stream to create.</dd>
+      <dt><var class="Ar" title="Ar">vhost</var></dt>
+      <dd>The name of the virtual host to create the super stream into.</dd>
+      <dt><var class="Ar" title="Ar">partitions</var></dt>
+      <dd>The number of partitions the super stream will have.</dd>
+      <dt><var class="Ar" title="Ar">routing-keys</var></dt>
+      <dd>Comma-separated list of routing keys.</dd>
+      <dt><var class="Ar" title="Ar">max-length-bytes</var></dt>
+      <dd>The maximum size of partition streams, example values: 20gb,
+        500mb.</dd>
+      <dt><var class="Ar" title="Ar">max-age</var></dt>
+      <dd>The maximum age of partition stream segments, using the ISO 8601
+          duration format, e.g. PT10M30S for 10 minutes 30 seconds, P5DT8H for 5
```

Comment:
Same comment as above - Also, I would avoid use latin abbreviations such as "e.g." in documentation, use the full word here "For example: " so I would rewrite this entire sentence to: 
"The maximum age of the partition stream segments that are using the ISO 8601
          duration format. For example: PT10M30S for 10 minutes and 30 seconds or P5DT8H for 5 days and 8 hours.